examples/qa_eval_custom_check.ipynb

```diff
@@ -0,0 +1,235 @@
+    "    name=f\"QA Scenario w/ Context- Webbizz\",\n",
+    "    seed_data=seed_data\n",
+    ")\n",
+    "scenario = okareo.create_scenario_set(scenario_set_create)\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Question Answer Model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import libraries\n",
+    "from openai import OpenAI\n",
+    "from datetime import datetime\n",
+    "\n",
+    "# Import Okareo's handler for OpenAI models\n",
+    "from okareo.model_under_test import OpenAIModel\n",
+    "\n",
+    "# Create an instance of the OpenAI client\n",
+    "client = OpenAI(api_key=OPENAI_API_KEY)\n",
+    "\n",
+    "# Define a function to call the OpenAI API\n",
+    "# This function will be used to query the specified OpenAI model\n",
+    "def get_turbo_summary(messages, model=\"gpt-3.5-turbo\", temperature=0, max_tokens=500):\n",
+    "  response = client.chat.completions.create(\n",
+    "    model=model,\n",
+    "    messages=messages,\n",
+    "    temperature=temperature, \n",
+    "    max_tokens=max_tokens,\n",
+    "  )\n",
+    "  return response.choices[0].message.content\n",
+    "\n",
+    "# Define a template for the user prompt\n",
+    "USER_PROMPT_TEMPLATE = \"Question: {input.question} Context: {input.context}\"\n",
+    "\n",
+    "# Define a template to prompt the model to provide a summary\n",
+    "ANSWER_CONTEXT_TEMPLATE = \"\"\"\n",
+    "You will be provided with context and a question.\n",
+    "Answer the question based on the context.\n",
+    "\"\"\"\n",
+    "\n",
+    "# Create an instance of the OpenAIModel class\n",
+    "# This class is used to interact with the OpenAI model using user and system prompts\n",
+    "openai_model = OpenAIModel(\n",
+    "        model_id=\"gpt-4o\",\n",
+    "        temperature=0,\n",
+    "        system_prompt_template=ANSWER_CONTEXT_TEMPLATE,\n",
+    "        user_prompt_template=USER_PROMPT_TEMPLATE,\n",
+    "    )\n",
+    "\n",
+    "# Define the name of the model with the current timestamp\n",
+    "mut_name=f\"OpenAI Answering Model - {datetime.now().strftime('%m-%d %H:%M:%S')}\"\n",
+    "\n",
+    "# Register the model to use in a test run\n",
+    "model_under_test = okareo.register_model(\n",
+    "    name=mut_name,\n",
+    "    model=openai_model,\n",
+    "    update=True\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from okareo.checks import ModelBasedCheck, CheckType\n",
+    "check = okareo.create_or_update_check(\n",
+    "    name=\"Relevance Check\",\n",
+    "    description=\"Relevance check for QA with context\",\n",
+    "    check=ModelBasedCheck(\n",
+    "        prompt_template=\"\"\"Rate the relevance of the answer on a scale of 1 to 5. The output should be only 1 number.\n",
```

Comment:
Can we start from g-eval relevance prompt and modify it with -summary +question?